pipenv/environments.py

```diff
@@ -299,12 +299,16 @@ def initialize(self):
 
         self.PIPENV_VENV_IN_PROJECT = os.environ.get("PIPENV_VENV_IN_PROJECT")
         if self.PIPENV_VENV_IN_PROJECT is not None:
-            self.PIPENV_VENV_IN_PROJECT = (
-                self.PIPENV_VENV_IN_PROJECT.lower() in _true_values
-            )
-        """If set, creates ``.venv`` in your project directory.
```

Comment:
Good good! Those `()` around the value were confusing, and I am glad you removed it.